src/main.ts

```diff
@@ -1,12 +1,16 @@
 import * as core from '@actions/core';
+import pino from 'pino';
 
+import {makeAxiosInstanceWithRetry} from './client';
 import {Build, Deployment, Emit, Status} from './emit';
 
 const BUILD = 'build';
 const DEPLOYMENT = 'deployment';
 const MODEL_TYPES = [BUILD, DEPLOYMENT];
 
 async function run(): Promise<void> {
+  const logger = pino({name: 'faros-client'});
```

Comment:
you might wanna prettify the first level here

Reply:
e.g. `pino({name: 'faros-client', prettyPrint: {levelFirst: true}}`

Reply:
and that might require another dependency `pino-pretty`

Reply:
good call!